docs/sources/configure/options.md

```diff
@@ -396,6 +396,16 @@ expression.
 If other selectors are specified in the same `services` entry, the processes to be
 selected need to match all the selector properties.
 
+| YAML             | Env var | Type                        | Default |
```

Comment:
To get rid of the linter complaint, change `Env var` by `Environment variable`

Reply:
There are already 54 mentions of `Env var` in the file. All tables include this one. Should I change it for all of them?

```
 grep -r "Env var" docs/sources/configure/options.md |wc -l
54
```

Reply:
I updated all of them in a separate commit

Reply:
Not needed! The linter only complains about the lines modified in the PR.

Reply:
I changed all of them in a separate commit. I think it will look weired if only one table header is different from all others. Let me know if I should drop the commit